geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionFunctionArgs.java

```diff
@@ -48,18 +52,18 @@
   private String keyConstraint;
   private String valueConstraint;
   private Boolean statisticsEnabled;
-  private RegionFunctionArgs.ExpirationAttrs entryExpirationIdleTime;
-  private RegionFunctionArgs.ExpirationAttrs entryExpirationTTL;
-  private RegionFunctionArgs.ExpirationAttrs regionExpirationIdleTime;
-  private RegionFunctionArgs.ExpirationAttrs regionExpirationTTL;
-  private RegionFunctionArgs.EvictionAttrs evictionAttributes;
+  private ExpirationAttrs entryExpirationIdleTime;
```

Comment:
The footprint of this class has changed.  the `serialVersionUID` must be updated.